examples/test/test_i_basic_encryption.py

```diff
@@ -11,19 +11,18 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 """Unit test suite for the Strings examples in the AWS-hosted documentation."""
-import os
-
 import botocore.session
 import pytest
 
 from ..src.basic_encryption import cycle_string
 from .examples_test_utils import get_cmk_arn
+from .examples_test_utils import static_plaintext
```

Comment:
We prefer to have all imports from the same module in the same import statement.